pkg/azuredisk/controllerserver.go

```diff
@@ -69,14 +69,17 @@ func (d *Driver) CreateVolume(ctx context.Context, req *csi.CreateVolumeRequest)
 		klog.Errorf("invalid create volume req: %v", req)
 		return nil, err
 	}
-	diskParams, err := azureutils.ParseDiskParameters(req.GetParameters())
+	params := make(map[string]string, len(req.GetParameters())+len(req.GetMutableParameters()))
+	for k, v := range req.GetParameters() {
+		params[k] = v
+	}
+	for k, v := range req.GetMutableParameters() {
```

Comment:
MutableParameters is only for `ControllerModifyVolume`, not for CreateVolume

Reply:
from the `CreateVolume` [spec](https://github.com/container-storage-interface/spec/blob/master/spec.md#createvolume) 
> The parameters field SHALL contain opaque volume attributes to be specified at creation time. The mutable_parameters field SHALL contain opaque volume attributes that are defined at creation time but MAY also be changed during the lifetime of the volume via a subsequent ControllerModifyVolume RPC. Values specified in mutable_parameters MUST take precedence over the values from parameters.

This is necessary to create volumes with the correct attributes when using VAC